metrics/transport/signalfx_test.go

```diff
@@ -15,8 +15,7 @@ func TestWriteToSFX(t *testing.T) {
 	require.NoError(t, err)
 	env := metrics.NewEnvironment(trans)
 	c := env.NewCounter("test.unittest.1", metrics.DimMap{"test": true})
-
-	require.NoError(t, c.Count(nil))
+	c.Count(nil)
```

Comment:
We should probably add the `assert.Fail` error handler like you have in other tests.